changes/4420.changed

```diff
@@ -0,0 +1 @@
+Update jobs migration docs to reflect alternate method of job params migration.
```

Comment:
```suggestion
Updated jobs migration docs to reflect alternate method of job params migration.
```

I wonder - should this probably be a `.added` rather than a `.changed`?